src/main/java/com/google/sps/servlets/RecipeServlet.java

```diff
@@ -149,6 +149,20 @@ public void doDelete(HttpServletRequest request, HttpServletResponse response)
   protected String getRecipeList(HttpServletRequest request, HttpServletResponse response) {
     String creatorToken = request.getParameter("token");
 
+    SortingMethod sortingMethod;
+    if (request.getParameter("sort") == null) {
```

Comment:
Should also check for invalid sorting method and page number here, as one can put arbitrary values for this in query params.

Reply:
Yup. Put in a try catch here to parse using `SortingMethod.valueOf` in 6dcf7cb, which should catch all invalid args to the sort parameter.